### ISSUE TITLE
[Scaling] Quests not scaling to level.

### ISSUE THREAD

**Issue by GiulianoMSA** [Project-AuroraWoW/Project-Aurora-Bug-Tracker#535]
Quests are not scaling to player level and are becoming gray and giving less experience before their usual level cap.
Gear rewards seem to be scaling properly.

1. Start a new character and go through the quests until you outlevel them.

**Expected behavior**
Quests should scale to player level until their cap(60 for 'vanilla' content for example)

**Comment by Madladdle** [comment]
Could you specify which zones?

**Comment by GiulianoMSA** [comment]
Well I'm pretty sure that was on all zones at the time I was leveling, but here is a list of the zones I did while leveling, where I saw quests becoming green and grey:
Durotar > Northern Barrens > Stonetalon Mountains > Southern Barrens > Dustwallow Marsh > Winterspring > Hellfire Peninsula > Shadowmoon Valley
I skipped Northrend and did only Mount Hyjal at cata level, and I saw some quests there becoming green(the initial ones that I couldn't complete, I did many from the Firelands chain so they were yellow).
I did only very few quests at the start of Panda and don't remember them being green by the time I left.
At WoD I did about half of the horde's initial zone(forgot name) and they started turning green when I got to 96+, so I went to Nagrand and finished leveling to 100 with yellow quests there.
